packages/wonder-blocks-popover/src/components/popover.js

```diff
@@ -118,6 +118,11 @@ type State = {|
      * Current popper placement
      */
     placement: Placement,
+
+    /**
+     * Popover Content ref
+     */
+    contentRef: RefObject<PopoverContent | PopoverContentCore>,
```

Comment:
It's better if this variable is declared as a class field declaration (or instance property).

e.g.
```js
class Foo {
    contentRef:.... = React.createRef();
  
   render() {}
}
```


```suggestion
    // somewhere at class level
    contentRef: RefObject<PopoverContent | PopoverContentCore> = React.createRef();
```